.github/workflows/release.yml

```diff
@@ -0,0 +1,57 @@
+name: Release
+
+permissions:
+  contents: write
+
+on:
+  push:
+    tags:
+      - mountpoint-s3-[0-9]+.*
+      - mountpoint-s3-[a-z]+-[0-9]+.*
+
+jobs:
+  create-release:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+    - uses: taiki-e/create-gh-release-action@v1
+      with:
+        prefix: mountpoint-s3(-[a-z]+)?
+        draft: true
+        title: "$prefix $version"
+        token: ${{ secrets.GITHUB_TOKEN }}
+
+  upload-assets:
+    name: Build and release on target (${{ matrix.target }})
+    runs-on: ${{ matrix.os }}
+    
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest]
```

Comment:
Let's not build on `ubuntu-latest`, because it changes when there's a new release. I think we want `ubuntu-20.04`, which is the oldest Ubuntu that GitHub Actions still offers.

Can you test that a binary built this way runs successfully on AL2?